templates/index.html

```diff
@@ -382,7 +382,7 @@ <h2>Lightweight workstation deployments</h2>
       <p>With the expansion of Kubeflow as a full ML toolkit, running it on desktops with up to 16Gb of RAM has become a painful experience.</p>
       <p>Kubeflow-lite is a lightweight version of Kubeflow with the ML services you typically need on your desktop, on a smooth experience.</p>
       <p>
-        <a href="https://jaas.ai/u/kubeflow-charmers/kubeflow-lite" class="p-link--external">Learn more</a>
+        <a href="https://jaas.ai/u/kubeflow-charmers/kubeflow-lite" class="p-link--external">Kubeflow-lite</a>
```

Comment:
```suggestion
        <a href="https://ubuntu.com/kubeflow/install" class="p-button--positive p-link--external">Install Kubeflow lite</a>
        <a href="https://jaas.ai/u/kubeflow-charmers/kubeflow-lite" class="p-link--external">Learn more about Kubeflow-lite</a>
```

Reply:
Updated the copy doc to match this change. https://docs.google.com/document/d/1Wvhl0yYV_w0BJsyZbQFKg_QV6RjMLbmQU3x0rsD2F4Q/edit#heading=h.mvjb0cu9g00d

Reply:
New PR here [https://github.com/canonical-web-and-design/charmed-kubeflow.io/pull/17](https://github.com/canonical-web-and-design/charmed-kubeflow.io/pull/17)